components/layout/page-section.tsx

```diff
@@ -0,0 +1,25 @@
+import { PropsWithChildren, ReactNode } from 'react';
+import { Box } from '@chakra-ui/react';
+
+export interface PageSectionProps {
+  backgroundColor?: 'green';
+}
+
+export const PageSection: React.FC<PropsWithChildren<PageSectionProps>> = ({
+  children,
+  backgroundColor: backgroundColorProp,
```

Comment:
Should pass sx?

Reply:
We don't use sx for this as its more of an outline component - we do however use it for the components inside it